libs/compute_cuda/include/hpx/compute/cuda/detail/launch.hpp

```diff
@@ -85,8 +85,8 @@ namespace hpx { namespace compute { namespace cuda { namespace detail {
         }
 
         template <typename DimType>
-        HPX_HOST_DEVICE static void call(target const& tgt, DimType gridDim,
-            DimType blockDim, fun_type f, args_type args)
+        HPX_HOST_DEVICE static void call(hpx::cuda::target const& tgt,
+            DimType gridDim, DimType blockDim, fun_type f, args_type args)
```

Comment:
While this is not introduced by this PR: could we please change the names of the arguments to conform to our naming conventions (all `lower_case_names`) while we touch this anyways?